From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@antonfrMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@alancameronwills can you work with @antonfrMSFT to fix the merge conflicts in the PR, or if you don't need it anymore please close it. Thanks.

Commit:
python-twin-how-to-configure initial draft

MGoedtel committed:
updated data security article and renamed it

MGoedtel committed:
removed sentence from intro para

MGoedtel committed:
Updated intro section

Commit:
iot-hub-firmware-update-python initial draft

Commit:
iot-hub-schedule-jobs-python initial draft

Commit:
iot-hub-schedule-jobs-csharp-csharp initial draft

Commit:
iot-hub-file-upload-python initial draft

Commit:
riskevents01

Commit:
riskevents01

Commit:
riskevents01

Commit:
riskevents01

Commit:
riskevents01

Commit:
riskevents01

Commit:
code revision, new screenshot

MGoedtel committed:
minor edits performed

MGoedtel committed:
updated per edit and feedback

MGoedtel committed:
new image for article

MGoedtel committed:
delete image file

MGoedtel committed:
resized image

barclayn committed:
making modification to the get started article to direct customers to azure monitor

barclayn committed:
minor editing to the intro

tynevi committed:
Update functions-cli-create-app-service-plan.md

tynevi committed:
fix mc

tynevi committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into acro-test-script-includes

tynevi committed:
fix mc

barclayn committed:
adding modifications to get started

barclayn committed:
fixing broken link

barclayn committed:
fixing table formatting issues and adding a warning about using siem connectors over azlog

barclayn committed:
adding the warning about the azure vm

barclayn committed:
adding azlog warning

Commit:
task 917: Add caching rules example and add screenshots

rwike77 committed:
fixed MSDN forum link

barclayn committed:
Final changes to azlog docs

ChristianWolf42 committed:
Event grid integration

ChristianWolf42 committed:
Fix links

barclayn committed:
fixing the faq

barclayn committed:
making modifications

Commit:
changes per @dominicbetts

Commit:
code update per @dominicbetts

barclayn committed:
fixing a note

barclayn committed:
Merge branch 'master' into azlog-warning

Commit:
changes per @dominicbetts

Commit:
code edit

v-geberr committed:
data extraction concept

v-geberr committed:
edits

barbkess committed:
updates

Commit:
changes per @dominicbetts

Commit:
changes per @dominicbetts

barbkess committed:
improvements

barbkess committed:
updates

ChristianWolf42 committed:
Fixing Acrolinx

ChristianWolf42 committed:
Fixing headlines, more Acrolinx

ChristianWolf42 committed:
Last Acrolinx fix

ChristianWolf42 committed:
Fixing author

ChristianWolf42 committed:
Last try in fixing author...

barbkess committed:
cross-link to overview

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into compute6

barbkess committed:
remove includes we aren't using

MikeDodaro committed:
News search SDK topic.  Small edits to web and image SDK topics.

MikeDodaro committed:
Add TOC link for News search SDK topic

MikeDodaro committed:
Video SDK java quickstart. Add to TOC. Small fix to Image SDK quickstart.

MikeDodaro committed:
Entity search Java quickstart.  Add to TOC.  Add main funtion to Web and Image quickstarts.

MikeDodaro committed:
Add main function to run documented methods.

MikeDodaro committed:
Add main method to run code samples.

MikeDodaro committed:
Take out live subscription key.

MikeDodaro committed:
Take out live subription key.

MikeDodaro committed:
Add dependencies tag to image file.

barbkess committed:
fix PR reviewer remarks

MikeDodaro committed:
Added import statements to Entity SDK example

MikeDodaro committed:
Add main function to Image SDK example

MikeDodaro committed:
Add main to web and image. Avoid conflicts

barclayn committed:
minor changes per reviewer

barclayn committed:
adding minor changes from reviewers

barclayn committed:
fixing formatting issues

carolinacmoravia committed:
M22873: Duplicated word: "virtual machine"

Hello, @jimdial,

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

rwike77 committed:
Update

rmca14 committed:
Merge pull request #32999 from MarkusVi/riskevents01

riskevents01

Commit:
Merge pull request #33640 from rwike77/patch-28

Update

Lisaco88 committed:
Merge pull request #33515 from rwike77/tocupdate

fixed MSDN forum link

v-geberr committed:
edits

v-geberr committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 0216-concept-data-extraction

Commit:
notes revision

v-geberr committed:
edits

PRMerger13 committed:
Merge pull request #33470 from tysonn/acro-test-script-includes

Acro test script includes

v-geberr committed:
edits

Commit:
Updated appservice ports

v-shils committed:
Merge pull request #32042 from msebolt/python-twin-how-to-configure

python-twin-how-to-configure initial draft

ChristianWolf42 committed:
Fixing blocking issues.

PRMerger20 committed:
Merge pull request #33636 from carolinacmoravia/patch-622

M22873: Duplicated word: "virtual machine"

Commit:
Updated for acrolinx hits

v-geberr committed:
edits

MGoedtel committed:
Updated article

curtand committed:
labeled feature as preview

MicrosoftGuyJFlo committed:
[Azure AD B2C] Update aka.ms links to HTTPS

barclayn committed:
image formatting problem

Lisaco88 committed:
Merge pull request #32602 from msebolt/iot-hub-firmware-update-python

iot-hub-firmware-update-python initial draft

v-geberr committed:
touch for samples build

v-geberr committed:
edits

barclayn committed:
image formatting issue

Commit:
Wups! I thought that was in the cert name!

curtand committed:
chNGED UNIFIED GROUPS TO Office 365 groups

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into compute6

Commit:
Merge pull request #33650 from MicrosoftGuyJFlo/HTTPtoHTTPSb2c

[Azure AD B2C] Update aka.ms links to HTTPS

barclayn committed:
trying to fix an image formatting problem

v-geberr committed:
edits

Lisaco88 committed:
Merge pull request #33642 from jeffgilb/jeff220

Updated appservice ports

PRMerger14 committed:
Merge pull request #33652 from v-geberr/0220-doc-touch-add-utterance-js

touch for samples build

MikeDodaro committed:
Fix titleSuffix

MightyPen committed:
Remove limitation about Azure Web App.

Email, from PM Dhruv Malik, received Tuesday 2018-02-20 12:05pm (Redmond WA USA time).
MightyPen = GeneMi.

v-geberr committed:
touch add utterances java

MGoedtel committed:
made updates per editor feedback

MikeDodaro committed:
Fix titleSuffix

barclayn committed:
one h2 not displaying properly

PRMerger15 committed:
Merge pull request #33651 from curtand/naming0220

labeled feature as preview

MikeDodaro committed:
Fix titleSuffix

v-geberr committed:
touch add utterances cs

vigneshcmsft committed:
Early events 

Added documentation about early events.

Lisaco88 committed:
Merge pull request #32630 from msebolt/iot-hub-schedule-jobs-python

iot-hub-schedule-jobs-python initial draft

v-shils committed:
Merge pull request #33529 from ChristianWolf42/SBtoEventGrid

Sb to event grid documentation

barbkess committed:
update best practices

Commit:
Merge pull request #33654 from MightyPen/patch-3

Remove limitation about Azure Web App.

rjagiewich committed:
Merge pull request #33609 from MikeDodaro/entity-sdk-java-quickstart

Entity search Java quickstart.  Add to TOC.  Add main funtion to Web …

v-geberr committed:
update to vs solution example

Ja-Dunn committed:
Merge pull request #32762 from msebolt/iot-hub-file-upload-python

iot-hub-file-upload-python initial draft

MikeDodaro committed:
Merge branch 'master' into video-sdk-java-quickstart

MikeDodaro committed:
Merge branch 'master' into news-sdk-java-quickstart

Commit:
Merge branch 'master' into iot-hub-schedule-jobs-csharp-csharp

v-geberr committed:
change date

Ja-Dunn committed:
Merge pull request #33411 from barclayn/azlog-warning

making modification to the get started article to direct customers to…

v-geberr committed:
touch ruby

MGoedtel committed:
fixed locale issue identified

MikeDodaro committed:
Update image-sdk-java-quickstart.md

rjagiewich committed:
Merge pull request #33574 from barbkess/compute6

Compute management overview

MikeDodaro committed:
Update image-sdk-java-quickstart.md

v-geberr committed:
h1 fix

Commit:
Merge pull request #33658 from v-geberr/0220-doc-touch-add-utterance-java

touch add utterances java

Brenduns committed:
Updates from PM for two topics

PRMerger17 committed:
Merge pull request #33662 from v-geberr/0220-doc-touch-add-utterance-ruby

touch ruby

Ja-Dunn committed:
Merge pull request #32498 from MGoedtel/UpdateLASecurity130

updated data security article and renamed it

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-geberr committed:
touch python

PRMerger14 committed:
Merge pull request #33655 from vigneshcmsft/patch-3

Early events

Ja-Dunn committed:
Merge pull request #33016 from msebolt/iot-hub-schedule-jobs-csharp-csharp

Iot hub schedule jobs csharp csharp

v-geberr committed:
touch php

Ja-Dunn committed:
Merge pull request #33500 from dksimpson/dean-917

task 917: Add caching rules example and add screenshots

Ja-Dunn committed:
Merge pull request #33570 from v-geberr/0216-concept-data-extraction

data extraction concept

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #33663 from Brenduns/tuesday-update

Updates from PM for two topics

Ja-Dunn committed:
Merge pull request #33597 from MikeDodaro/news-sdk-java-quickstart

News sdk java quickstart

MikeDodaro committed:
Merge branch 'master' into video-sdk-java-quickstart

rjagiewich committed:
Merge pull request #33657 from v-geberr/0220-doc-touch-add-utterance-cs

0220 doc touch add utterance cs

v-alje committed:
Merge pull request #33669 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #33599 from MikeDodaro/video-sdk-java-quickstart

Video SDK java quickstart. Add to TOC. Small fix to Image SDK quickst…

PRMerger19 committed:
Merge pull request #33666 from v-geberr/0220-doc-touch-add-utterance-python

touch python

PRMerger19 committed:
Merge pull request #33667 from v-geberr/0220-doc-touch-add-utterance-php

touch php
